### DISCUSSION TITLE
How about direct call syscall on Linux, remove the deps on libc on Linux?

### DISCUSSION THREAD

**Discussion by lygstate** [vlang/v#11677] [Ideas]
On Linux, there is different libc such as musl and glibc, and diferent libc are not compt, but syscall are relative stable.
https://filippo.io/linux-syscall-table/

**Comment by spytheman** [comment]
There was preliminary work for this behind the `-freestanding` flag, but it does not work currently.
See vlib/builtin/bare/ if you want to work on it.

**Comment by lygstate** [comment]
Good to see that:)
